### PR TITLE
Made them into code (README)

### PR DESCRIPTION
**Pull request by czipperz**

### CODE REVIEW

**Comment by npostavs** [comment]
If you're modifying that, perhaps also update the code snippets to the new naming style: `yas-` instead of `yas/`.

**Comment by czipperz** [comment]
@npostavs did

File: README.md [hunk 1]
```diff
@@ -5,24 +5,24 @@ This repository contains the official snippets for [yasnippet](http://github.com
...
-3. M-x yas/reload-all to activate them
+3. `M-x yas-reload-all` to activate them
 
 # Contributing
 
 This repository has now become the default snippets repository (as a submodule) in yasnippet.
 So if you have any useful snippets for any language or framework please feel free to contribute.
 
-To study the current snippets I suggest to use M-x yas/describe-tables
+To study the current snippets I suggest to use `M-x yasdescribe-tables`
```
Comment (npostavs) [inline README.md]: missed a `-` here

**Comment by czipperz** [comment]
I feel like that comment was pure bureaucracy

**Comment by npostavs** [comment]
> I feel like that comment was pure bureaucracy

I don't know what you mean? Would you have preferred I didn't post the comment? I think it's nicer to avoid typos in documentation so I pointed the typo out...

**Comment by czipperz** [comment]
Oh no, it was good to point out. I was just saying that you commenting about a simple spelling mistake and being unable to fix it was amusing

**Comment by npostavs** [comment]
Oh that's good then, my first impression was that you were pissed off by the bureaucracy. Maybe I should take a cue from your handle and read everything you post in a "chipper" tone? :)

**Comment by AndreaCrotti** [comment]
Nice thanks to both!

**Comment by czipperz** [comment]
@npostavs nice

**Comment by czipperz** [comment]
@AndreaCrotti btw, if I come across a language that isn't included here, should I be adding it on? I'm wondering what the design is going for

**Pull request merged**